gadget/validate.go

```diff
@@ -187,11 +194,11 @@ func ensureVolumeRuleConsistency(state *validationState, model Model) error {
 }
 
 func ensureSeedDataLabelsUnset(state *validationState) error {
```

Comment:
the name no longer fits here, as we allow labels to be set now (although only using the implicit label). So maybe `ensureSeedDataLabelsImplicit`?

Reply:
thanks for spotting this, given the new behavior I think check* something name is better, will change